web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -4,9 +4,9 @@ title: Migration from 0.11.X to 0.12.X
 
 import { EmailPill, UsernameAndPasswordPill, GithubPill, GooglePill } from "./auth/Pills";
 
-:::note The latest version of Wasp is 0.13.X
+:::note The latest version is a few versions away
```

Comment:
Weird sentence IMO. If it wasn't for the old text, I wouldn't know what we wanted to say. Maybe:
```
Wasp has had several releases since 0.12.0, and its current version is well ahead of 0.12.0.

To fully migrate...
````

Reply:
Updated it to make more sense 👍